### PR TITLE
[README.md] add document to convert from stl to urdf, eus.

### PR DESCRIPTION
**Pull request by mmurooka**

### CODE REVIEW

**Comment by mmurooka** [comment]
https://github.com/euslisp/jskeus/pull/248

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+    <collision>
+     <origin xyz="0 0 0" rpy="0 0 0"/>
+     <geometry>
+       <mesh filename="file:///home/leus/my_model.stl" />
+     </geometry>
+    </collision>
+  </link>
+</robot>
+```
+=> my_model.urdf is generated.
```
Comment (knorth55) [inline README.md]: 'generated' seems 'automatically generated', but this urdf is written manually, is it correct?
if so, verb 'generate' seems not proper for me.

Comment (mmurooka) [inline README.md]: Fixed.

Comment (knorth55) [inline README.md]: thx

**Comment by mmurooka** [comment]
Travis test passed. Cloud you merge this if no problem?

**Comment by k-okada** [comment]
do we keep maintain these three path?
1) stl->urdf->collada->eus, 
2) eusassmp, 
3) stl->eus

**Comment by mmurooka** [comment]
>  1) stl->urdf->collada->eus, 

stl -> urdf is not conversion but manually generating new file, so we need no maintenance.
urdf -> collada -> eus is necessary for ROS robot (f.g. pr2, baxter).

> 3) stl->eus 

would be useful but どちらでもいいと思います．

**Comment by k-okada** [comment]
2016-10-20 20:06 GMT+09:00 Masaki Murooka <notifications@github.com>:

> would be useful but どちらでもいいと思います．
>
>
please write so on jskeus pull request

--
◉ Kei Okada

**Comment by mmurooka** [comment]
referred this comment in jskeus PR.

**Comment by mmurooka** [comment]
Travis test passed. Should wait until https://github.com/euslisp/jskeus/pull/248 is merged?

**Pull request merged**